src/hooks/use-drag-and-drop-file.ts

```diff
@@ -36,10 +36,26 @@ export function useDragAndDropFile< T extends HTMLElement >( {
 			if ( event.dataTransfer.files.length === 1 ) {
 				onFileDrop( event.dataTransfer.files[ 0 ] );
 			}
+			setIsDraggingOver( false );
+		};
+		const handleDragEnd = ( event: DragEvent ) => {
+			event.preventDefault();
+			setIsDraggingOver( false );
+		};
+
+		const node = dropRef.current;
```

Comment:
What do you think about using dropNode, dropZone, dropElement instead of just node?
We could declare it at the beginning of the useEffect, and use it in the rest of the function.

```suggestion
		const dropNode = dropRef.current;
```

Reply:
Done in https://github.com/Automattic/studio/pull/378/commits/cdcac1b629a7b1b877725714021416e0368fb347
As mentioned in another comment, I also removed the `dragend`